cmd/server-main.go

```diff
@@ -277,7 +281,93 @@ func mergeServerCtxtFromConfigFile(configFile string, ctxt *serverCtxt) error {
 		ctxt.SFTP = append(ctxt.SFTP, fmt.Sprintf("ssh-private-key=%s", cf.Options.SFTP.SSHPrivateKey))
 	}
 
-	ctxt.Layout, err = buildDisksLayoutFromConfFile(cf.Pools)
+	ctxt.Layout, err = buildDisksLayoutFromConfFile(cf.EnableExpandPools, cf.Pools, false)
+
+	if ctxt.EnableExpandPools {
```

Comment:
Keep to the left.

`if ctxt.EnableExpandPools {` should be `if !ctxt.EnableExpandPools { return }`....